pkg/opssight/creater.go

```diff
@@ -282,42 +282,42 @@ func (ac *Creater) addRegistryAuth(opsSightSpec *opssightapi.OpsSightSpec) {
 }
 
 func (ac *Creater) postDeploy(spec *SpecConfig, namespace string) error {
-	// Need to add the perceptor-scanner service account to the privileged scc
-	if ac.osSecurityClient != nil {
-		scannerServiceAccount := spec.ScannerServiceAccount()
-		perceiverServiceAccount := spec.PodPerceiverServiceAccount()
-		serviceAccounts := []string{fmt.Sprintf("system:serviceaccount:%s:%s", namespace, perceiverServiceAccount.GetName())}
-		if !strings.EqualFold(spec.opssight.Spec.ScannerPod.ImageFacade.ImagePullerType, "skopeo") {
-			serviceAccounts = append(serviceAccounts, fmt.Sprintf("system:serviceaccount:%s:%s", namespace, scannerServiceAccount.GetName()))
-		}
-		return util.UpdateOpenShiftSecurityConstraint(ac.osSecurityClient, serviceAccounts, "privileged")
-	}
-
 	// Create Perceptor model Route on Openshift
-	if strings.ToUpper(spec.opssight.Spec.Perceptor.Expose) == "OPENSHIFT" && ac.routeClient != nil {
+	if strings.EqualFold(spec.opssight.Spec.Perceptor.Expose, "OPENSHIFT") && ac.routeClient != nil {
```

Comment:
Strings like this really should be constants.  I would put in an api package and have synopsysctl import that package and set the value using a constant.  Otherwise there is room for stuff to get out of sync.  It also means comparisons like this become a simple == instead of EqualFold